ui/src/styles/_fliptiles.scss

```diff
@@ -0,0 +1,34 @@
+.flip-container {
+  perspective: 1000px; /* Remove this if you don't want the 3D effect */
+
+  /* This container is needed to position the front and back side */
+  .flipper {
+    transition: transform 0.4s;
+    transform-style: preserve-3d;
+
+    /* Position the front and back side */
+    .front,
+    .back {
+      position: absolute;
+      -webkit-backface-visibility: hidden; /* Safari */
+      backface-visibility: hidden;
+    }
+
+    /* Style the front side */
+    .front {
+      z-index: 2;
```

Comment:
whats the z index doing here? wouldnt this just always put this element on top then

Reply:
ensures that the front side always gets shown by default. .back has a z-index of 1